app-admin/prelude-manager/files/prelude-manager.initd

```diff
@@ -0,0 +1,25 @@
+#!/sbin/runscript
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+BIN_MANAGER=/usr/bin/prelude-manager
+PID_MANAGER=/run/prelude-manager/prelude-manager.pid
+
+depend() {
+  need net
+}
+
+start() {
+  ebegin "Starting prelude-manager"
```

Comment:
Insert `checkpath -d -m 0700 /run/prelude-manager` here to make sure that the pid directory is properly created. I chose mode `0700` because the files are created by prelude-manager with the same mode.